frontend-project/src/components/Table/ChannelName.tsx

```diff
@@ -1,7 +1,7 @@
 import React, { FC, useEffect, useState } from 'react';
 import { Spin } from 'antd';
 
-import { fetchChannel } from '@/services/channel';
+import { fetchChannel } from '@/services/channels';
```

Comment:
raczej kierowałbym się tutaj w stronę store, żeby w przyszłości dało się łatwiej tego reuseować

Reply:
trochę nad tym siedziałem znalazłem dwa rozwiązania: 
1) ściągnąć wszystkie kanały do stora (tutaj nie korzystamy z paginacji na backendzie)
2) efekt odpala reducera, ale również zwraca ściągnięte dane, tabela nie korzysta z danych z stora ale z tego co zwrócił efekt - nie wiem, czy to jest odpowiedni sposób na poradzenie sobie z tym zagadnieniem.

Reply:
dobra, wykombinowałem rozwiązanie zaraz przygotuje PR

Reply:
zmiana dodana w innym PR #472